lib/init.js

```diff
@@ -0,0 +1,88 @@
+'use babel';
+
+import { parse, exec } from 'atom-linter';
+import { dirname, extname } from 'path';
+
+const isWindows = (/^win/).test(process.platform);
+const pattern = '.+:(?<line>\\d+):(?<col>\\d+)-(?<colEnd>\\d+):' +
+  ' \\((?<type>[EW])\\d+\\) (?<message>.*)';
+
+const checkedAppend = function checkedAppend(parameters, opt, args) {
```

Comment:
Make this name more specific, something like `checkedOption`, since we are specifically conditionally adding an option.